### PR TITLE
feat(openchallenges): add link to our Discord server to the OC navbar

### LINKED ISSUE #2278: [Feature] Add link to our Discord server to the OC navbar
### What product(s) is this feature for?

OpenChallenges

### Description

Add a link/button to the right side of the navbar that opens our Discord server.

For example:

image

Our navbar is relatively empty for now so we could add a label to the button

### Anything else?

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by tschaffter**
Closes #2278

## Changelog

- Add a Discord button to the OC navbar with permanent invite link

## Preview

image

### Preview design

image

### CODE REVIEW

**Comment by vpchung** [comment]
Nice!  What do you think about integrating a [Discord widget](https://github.com/TitanEmbeds/Titan) into the OC?  That way, non-Discord users can still participate and communicate in the channels (as guests) without needing to create a Discord account.

File: libs/openchallenges/ui/src/lib/discord-button/discord-button.component.ts [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+@Component({
+  selector: 'openchallenges-discord-button',
+  standalone: true,
+  imports: [CommonModule, MatButtonModule],
+  templateUrl: './discord-button.component.html',
+  styleUrls: ['./discord-button.component.scss'],
+})
+export class DiscordButtonComponent {
+  @Input({ required: false }) label = 'Discord';
```
Comment (vpchung) [inline libs/openchallenges/ui/src/lib/discord-button/discord-button.component.ts:13]: Could we use a more descriptive label here?  e.g. "Contact us on Discord"

**Review by vpchung — APPROVED** [review]
Thanks! One minor suggestion.

File: libs/openchallenges/ui/src/lib/discord-button/discord-button.component.ts (hunk 1 above)
Comment (tschaffter) [reply libs/openchallenges/ui/src/lib/discord-button/discord-button.component.ts:13]: "Join us on Discord"? (a bit shorter - I will post a preview soon)

**Comment by tschaffter** [comment]
Arg, I posted the preview but forgot to push the update. I will open another PR to finalize the PR.

**Pull request merged**